front/lib/api/assistant/actions/dust_app_run.ts

```diff
@@ -247,108 +228,56 @@ export async function* runDustApp(
     throw new Error("Unexpected unauthenticated call to `runDustApp`");
   }
 
-  const actionConfig = deprecatedGetFirstActionConfiguration(configuration);
-
-  if (!isDustAppRunConfiguration(actionConfig)) {
-    throw new Error("Unexpected action configuration received in `runDustApp`");
-  }
-
-  if (owner.sId !== actionConfig.appWorkspaceId) {
-    yield {
-      type: "dust_app_run_error",
-      created: Date.now(),
-      configurationId: configuration.sId,
-      messageId: agentMessage.sId,
-      error: {
-        code: "dust_app_run_workspace_error",
-        message:
-          "Runing Dust apps that are not part of your own workspace is not supported yet.",
-      },
-    };
-    return;
-  }
-
-  const app = await getApp(auth, actionConfig.appId);
+  const app = await getApp(auth, actionConfiguration.appId);
   if (!app) {
     yield {
       type: "dust_app_run_error",
       created: Date.now(),
       configurationId: configuration.sId,
       messageId: agentMessage.sId,
       error: {
-        code: "dust_app_run_app_error",
-        message: `Failed to retrieve Dust app ${actionConfig.appWorkspaceId}/${actionConfig.appId}`,
+        code: "dust_app_run_parameters_generation_error",
```

Comment:
The previous error code had more sense, no?

Reply:
It's does not contain `error` in the code which is off our convention so far for all these errors codes + this is not an app run error but an error related to parameters generation.